### DISCUSSION TITLE
Marketplace - Links in browser's Dark Mode not well visible

### DISCUSSION THREAD

**Discussion by babsgosgens** [gameserverapp/Platform#998] [Suggestions]
See below image, the text color for links is not standing out well against a dark background, and is not consistent with the Dashboard UI. My suggestion is to change the color to Blue (at least in the context of the Dashboard). 

https://dash.gameserverapp.com/marketplace/market/shop_pack_group/4

![image](https://github.com/user-attachments/assets/4f4dce27-43ee-45bf-89e1-c0f8cd5547ed)

**Comment by mvaessen** [comment]
This was fixed on a previous update. Thanks for the feedback!
